links.json

```diff
@@ -2,6 +2,7 @@
     "$schema": "https://json-schema.org/draft/2019-09/hyper-schema",
     "$id": "https://json-schema.org/draft/2019-09/links",
     "title": "Link Description Object",
+
```

Comment:
intentional?

Reply:
Yeah, it's the same with other schema files.
@handrews why do we do this?

Reply:
Readability. This is fine and intended.

Reply:
I was inquiring about the extra whitespace.